src/style.css

```diff
@@ -1 +1,57 @@
 /* Styles go here */
+body {
+  margin: 0;
+}
+
+.container {
+  margin: 0;
+  display: flex;
+}
+
+.logo {
+  margin-top: 10px;
+  margin-left: 50px;
+  object-fit: none;
+  object-position: top;
+}
+
+.navigation {
+  width: 100vw;
+  font-family: "Roboto", sans-serif;
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+.list {
+  height: 60px;
+  line-height: 60px;
+  margin: 0 20px 0 0;
+  list-style-type: none;
+  cursor: pointer;
+}
+
+.nav_link {
```

Comment:
Following to the BEM names convention block element should be `nav__link` instead of `nav_link`